src/main/java/com/targomo/client/api/pojo/RoutingOptions.java

```diff
@@ -51,10 +51,19 @@ public class RoutingOptions {
 
     @Override //overriding hash code since the has would change with each execution for enums (travelType)
     public int hashCode(){
-        return Objects.hash(maxEdgeWeight, travelType.getKey(), travelTimeFactors, reverse, rushHour, time, date, frame, arrivalOrDepartureDuration,
+        return Objects.hash(maxEdgeWeight, travelType.ordinal(), travelTimeFactors, reverse, rushHour, time, date, frame, arrivalOrDepartureDuration,
```

Comment:
was there an issue with the key?

Reply:
it was working as well. It just to be more coherent with other part of the code where I use `ordinal()`